index.html

```diff
@@ -21,21 +21,31 @@
   <!-- The header section for mobile -->
   <header>
     <a href="#" class="logo">Grace Abike</a>
-    <nav class="mobile">
-      <img src="./imgs/Menumobile.png" alt="menu">
-    </nav>
+    <div id="hamburger-icon" class="mobile" onclick="toggleMenu(this)">
+      <div id="bar1" class="bar"></div>
+      <div id="bar2" class="bar"></div>
+      <div id="bar3" class="bar"></div>
+      <div id="mobileNav" class="overlay mobile mobile-menu">
+        <div class="overlay-content">
+          <a class="bar-link1" href="#main-section">Portfolio <span class="heavy-right">&#x276F;</span></a>
+          <a class="bar-link2" href="#work">About <span class="heavy-right">&#x276F;</span></a>
```

Comment:
- [ ] Kindy ensure your links are correctly done, the about button should not link to your projects as they are your portfolio, kindly check on it.

Reply:
@danmainah thanks for your comments I really appreciate.
Please can you show me or explain to me better where it's required not to link `About`

<img width="572" alt="image" src="https://user-images.githubusercontent.com/88346506/188496652-c4f68817-2c3f-4cff-9eec-9d5b6221d555.png">